sql/moz-fx-data-shared-prod/firefox_ios_derived/feature_usage_metrics_v1/query.sql

```diff
@@ -0,0 +1,444 @@
+-- Query for firefox_ios_derived.feature_usage_metrics_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+DECLARE start_date DATE DEFAULT "2021-01-01";
+
+DECLARE end_date DATE DEFAULT current_date;
+
+WITH dau_segments AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    COUNT(DISTINCT client_info.client_id) AS dau
+  FROM
+    `mozdata.firefox_ios.metrics`
```

Comment:
```suggestion
    firefox_ios.metrics
```

In this case we should aim to query the table in the same project as where we are writing the data. Omitting the project_id will default to the default project set in the processing environment. Unless the source table specifically lives only in a specific project please just use `dataset.table`.